src/libpanic_unwind/lib.rs

```diff
@@ -41,7 +41,7 @@ cfg_if::cfg_if! {
     if #[cfg(target_os = "emscripten")] {
         #[path = "emcc.rs"]
         mod real_imp;
-    } else if #[cfg(target_arch = "wasm32")] {
+    } else if #[cfg(any(target_arch = "wasm32", target_os = "none"))] {
```

Comment:
Could this perhaps be inverted to have `dummy.rs` be the fallback instead of the `gcc.rs` file?

Reply:
Sounds good.  There are a few new platforms which now use the dummy path:

- `i686-unknown-uefi` and `x86_64-unknown-uefi`  (pretty certain these do not support unwinding)
- `avr-unknown-unknown` (claims to support unwinding, but @shepmaster claims that is wrong)
- `mipsel-sony-psp` (claims to support unwinding, also verified with jonas-schievink also says this probably doesn't support unwinding)

I presume these were no_std platforms (sometimes it is hard to find this information).

Reply:
Avr is definitely no-std. core only